pkg/kubectl/cmd/cmd.go

```diff
@@ -235,6 +236,7 @@ Find more information at https://github.com/kubernetes/kubernetes.`,
 	cmds.AddCommand(NewCmdVersion(f, out))
 	cmds.AddCommand(NewCmdExplain(f, out))
 	cmds.AddCommand(NewCmdConvert(f, out))
+	cmds.SetOutput(os.Stdout)
```

Comment:
cmd.SetOutput(out) will dump usage errors from cobra to stdout.  Why would
we want that to do that?

On Tue, May 24, 2016 at 9:00 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/kubectl/cmd/cmd.go
> <https://github.com/kubernetes/kubernetes/pull/26077#discussion_r64385536>
> :
>
> > @@ -235,6 +236,7 @@ Find more information at https://github.com/kubernetes/kubernetes.` <https://github.com/kubernetes/kubernetes.>,
> >  	cmds.AddCommand(NewCmdVersion(f, out))
> >  	cmds.AddCommand(NewCmdExplain(f, out))
> >  	cmds.AddCommand(NewCmdConvert(f, out))
> > +	cmds.SetOutput(os.Stdout)
>
> I could change it to cmds.SetOutput(out) if that's what you meant :)
>
> Yes, thanks
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26077/files/abc4e0046bd11952535c7773e37f8c43100051b2#r64385536>
>